### PR TITLE
Update default value of automatic security patches on AzLinux GPU node pools

### PR DESCRIPTION
**Pull request by sdesai345**

### CODE REVIEW

File: articles/aks/gpu-cluster.md [hunk 1]
```diff
@@ -26,7 +26,11 @@ To view supported GPU-enabled VMs, see [GPU-optimized VM sizes in Azure][gpu-sku
 
 ## Limitations
 
-* If you're using an Azure Linux GPU-enabled node pool, automatic security patches aren't applied, and the default behavior for the cluster is *Unmanaged*. For more information, see [auto-upgrade](./auto-upgrade-node-image.md).
+* If you're using an Azure Linux GPU-enabled node pool, refer to your current AKS API version for the default behavior of automatic security patches on your cluster.
```
Comment (allyford) [inline articles/aks/gpu-cluster.md:29]: Keep the part about: automatic security patches aren't applied

Comment (sdesai345) [inline articles/aks/gpu-cluster.md:29]:
```suggestion
* If you're using an Azure Linux GPU-enabled node pool, automatic security patches aren't applied. Refer to your current AKS API version for the default behavior of node OS upgrade channel on your cluster.
```

File: articles/aks/gpu-cluster.md [hunk 2]
```diff
@@ -26,7 +26,11 @@ To view supported GPU-enabled VMs, see [GPU-optimized VM sizes in Azure][gpu-sku
 
 ## Limitations
 
-* If you're using an Azure Linux GPU-enabled node pool, automatic security patches aren't applied, and the default behavior for the cluster is *Unmanaged*. For more information, see [auto-upgrade](./auto-upgrade-node-image.md).
+* If you're using an Azure Linux GPU-enabled node pool, automatic security patches aren't applied. Refer to your current AKS API version for the default behavior of node OS upgrade channel on your cluster.
```
Comment (allyford) [inline articles/aks/gpu-cluster.md:29]:
```suggestion
* If you're using an Azure Linux GPU-enabled node pool, automatic security patches aren't applied. Refer to your current AKS API version for the default behavior of node OS upgrade channel.
```

File: articles/aks/gpu-cluster.md [hunk 3]
```diff
@@ -26,7 +26,11 @@ To view supported GPU-enabled VMs, see [GPU-optimized VM sizes in Azure][gpu-sku
...
+
+> [!NOTE]
+> For AKS API version 2023-06-01 or later, the default value for the cluster is *NodeImage*. For previous versions, the default value is *None*. To learn more, see [auto-upgrade](./auto-upgrade-node-image.md).
```
Comment (allyford) [inline articles/aks/gpu-cluster.md:32]:
```suggestion
> For AKS API version 2023-06-01 or later, the default channel for node OS upgrade is *NodeImage*. For previous versions, the default channel is *None*. To learn more, see [auto-upgrade](./auto-upgrade-node-image.md).
```

**Review by allyford — APPROVED** [review]
Approved with changes

**Pull request closed without merging**